safe/gui/widgets/dock.py

```diff
@@ -611,6 +622,22 @@ def block_signals(self):
         self.exposure_layer_combo.blockSignals(True)
         self.hazard_layer_combo.blockSignals(True)
 
+    def debug_group_toggled(self):
+        """Helper to set a color on the debug button to know if it's debugging.
+
+        If debug analysis is true or if rounding is disabled, it will be
+        orange.
+        """
+        use_debug = self.use_debug_action.isChecked()
+        setting('use_debug_analysis', use_debug)
```

Comment:
set expected type to `bool`

Reply:
no, thanks it's a mistake. It must be 'set_setting'

Reply:
ah yes, of course. That's why there is no assignment.